### DISCUSSION TITLE
UCD log - Interaction Design guidance

### DISCUSSION THREAD

**Discussion by nhsbsa-sthil** [nhsbsa/nhsbsa-digital-playbook#235] [UCD]
Like the other UCD disciplines ([User Research](https://github.com/nhsbsa/nhsbsa-digital-playbook/discussions/231), [Service Design](https://github.com/nhsbsa/nhsbsa-digital-playbook/discussions/243) and [Content Design](https://github.com/nhsbsa/nhsbsa-digital-playbook/discussions/250)) the Senior Interaction Design team are putting together specific guidance on how to fill out their section of a UCD log.

A UCD log makes it easy to explain why a service is the way it is to various people including:

1. Stakeholders
2. Project team
3. New starters
4. Anyone who might be interested!

**Comment by nhsbsa-sthil** [comment]
@Tom-Leeds @nhsbsa-dean @JoshBSA Created this to capture the work currently happening...

Current document:

https://nhsbsauk-my.sharepoint.com/:w:/g/personal/thole_nhsbsa_nhs_uk/Ed42s4AdPalGqjjnyvtIN3EBMrGoRhklyZYAtSn1j9ifjw?e=4gNKA7

**Comment by nhsbsa-deaco** [comment]
New word document for writing up the guidance to go on the playbook, this is a first draft after work has been done on the previous document which held ideas. 

https://nhsbsauk-my.sharepoint.com.mcas.ms/:w:/g/personal/deaco_nhsbsa_nhs_uk/ERXCtr42r6pHv5_25oNVq7UBHGahz4zyTS_tiZtnVm4CsA?e=0GNecc

**Reply by nhsbsa-deaco** [reply]
Draft v1

Structure being followed for this is:

1. Introduction

2. Sample file structure 

3. Headings to structure your UCD log
3.1. Why we did this work
3.2. What out ideas were
3.3. How we tested them and what we found
3.4. What we will do next

4. Resources

**Comment by nhsbsa-deaco** [comment]
Email has been sent out to all Interaction Designers to collate feedback on the guide

**Comment by nhsbsa-deaco** [comment]
We are now into the 4th week of January and we have had 1 response to the form set up (this year - 5 in total) to collate all feedback from designers on the interaction design guidance, I think its fair to say we just go ahead with the guidance we have now, which is a shame because it would have been great to receive a good collection of feedback however I've asked enough times now.

**Reply by nhsbsa-deaco** [reply]
I will share the feedback on the IxD/SD guidance here shortly

**Comment by nhsbsa-deaco** [comment]
In total we have had 7 responses to the feedback form.

![Interaction Service Design Guidance Feedback](https://github.com/nhsbsa/nhsbsa-digital-playbook/assets/51478242/5defafd3-001f-4101-92ec-f7c78a36c7c5)

Excel document of the feedback:
[Draft Guide for Design Logs (Feedback)(1-7) (1).xlsx](https://github.com/nhsbsa/nhsbsa-digital-playbook/files/14054007/Draft.Guide.for.Design.Logs.Feedback.1-7.1.xlsx)

**Comment by nhsbsa-deaco** [comment]
Interaction Design Blank Template - Ready for pilot

[Interaction Design UCD Log Template.docx](https://github.com/nhsbsa/nhsbsa-digital-playbook/files/14534931/Interaction.Design.UCD.Log.Template.docx)
